docs/azure_arc_jumpstart/azure_arc_k8s/developer/dapr/dapr_statestore/_index.md

```diff
@@ -0,0 +1,205 @@
+---
+type: docs
+title: "Deploy an application using the Dapr cluster extension for Azure Arc-enabled Kubernetes cluster"
+linkTitle: "Deploy an application using the Dapr cluster extension for Azure Arc-enabled Kubernetes cluster"
+weight: 1
+description: >
+---
+
+## Deploy an application using the Dapr cluster extension for Azure Arc-enabled Kubernetes cluster
+
+The following Jumpstart scenario will guide you on how to set up [Dapr](https://dapr.io/) for the Azure Arc-enabled Kubernetes cluster. In this scenario, you will enable the Dapr by deploying the Dapr extension on your Azure Arc-enabled Kubernetes cluster. You will also be deploying a Python application that generates messages and a Node application that consumes and persists them on Azure Cache for Redis.
+
+Dapr is a portable, event-driven runtime that makes it easy for any developer to build resilient, stateless, and stateful applications that run on the cloud and edge and embraces the diversity of languages and developer frameworks. Leveraging the benefits of a sidecar architecture, Dapr helps you tackle the challenges that come with building microservices and keeps your code platform agnostic.
+
+> **NOTE: This guide assumes you already deployed a Kubernetes cluster and connected it to Azure Arc. If you haven't, this repository offers you a way to do so in an automated fashion for various [Kubernetes distributions](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/).**
+
+## Prerequisites
+
+- Clone the Azure Arc Jumpstart repository
+
+    ```shell
+    git clone https://github.com/microsoft/azure_arc.git
+    ```
+
+- As mentioned, this scenario starts at the point where you already have a connected Kubernetes cluster to Azure Arc.
+
+    ![Existing Azure Arc-enabled Kubernetes cluster](./01.png)
```

Comment:
Don't update for this scenario but for future scenarios, let's keep the standard of  cluster resource name to "Azure-Arc-Demo"

Reply:
Updated